readthedocs/projects/models.py

```diff
@@ -251,7 +251,10 @@ class Project(models.Model):
     show_version_warning = models.BooleanField(
         _('Show version warning'),
         default=False,
-        help_text=_('Show warning banner in non-stable nor latest versions.'),
+        help_text=_(
+            'Show a warning banner with a link to the stable version on not'
+            ' stable versions, except for latest.'
```

Comment:
Here also. I'd like to improve this sentence. It's really hard to parse and understand the last part: "the stable version on not stable versions, except for latest.". After reading it three times I think I understand what it means, but it was hard to get it.

Maybe splitting it in multiple sentences like:

    Show a warning banner with a link to the stable version.
    The banner is shown in all the versions, but in latest and stable.